### PR TITLE
Status: Better error message display

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

File: hawk/app/models/cib.rb [hunk 1]
```diff
@@ -62,7 +63,7 @@ def meta
         else
           :ok
         end
-      elsif errors.length == 1 and not @crm_config[:stonith_enabled]
+      when !@crm_config[:stonith_enabled]
```
Comment (tboerger) [inline hawk/app/models/cib.rb]: Ignore this

File: hawk/app/models/cib.rb [hunk 2]
```diff
@@ -49,7 +49,8 @@ def meta
       struct.version = crm_config[:dc_version]
       struct.stack = crm_config[:cluster_infrastructure]
 
-      struct.status = if errors.empty?
+      struct.status = case
+      when errors.empty?
```
Comment (tboerger) [inline hawk/app/models/cib.rb]: Ignore this

File: hawk/app/models/cib.rb [hunk 3]
```diff
@@ -80,6 +81,119 @@ def sim?
     id != 'live'
   end
 
+  def status(minimal = false)
```
Comment (tboerger) [inline hawk/app/models/cib.rb]: Ignore this

File: hawk/app/assets/stylesheets/application/_content.scss [hunk 4]
```diff
@@ -70,7 +70,8 @@
     margin: 20px 0px 20px 0px;
     font-size: 2em;
 
-    .fa.page {
+    .fa.page,
```
Comment (tboerger) [inline hawk/app/assets/stylesheets/application/_content.scss]: Ignore this

File: hawk/app/models/cib.rb [hunk 5]
```diff
@@ -769,7 +897,10 @@ def initialize(id, user, use_file = false)
     @op_defaults = Hashie::Mash.new Hash[@op_defaults.map {|k,v| [k.to_s.underscore.to_sym, v]}]
 
     if not @crm_config[:stonith_enabled]
-      error _("STONITH is disabled. For normal cluster operation, STONITH is required.")
+      error(
+        _("STONITH is disabled. For normal cluster operation, STONITH is required."),
```
Comment (tboerger) [inline hawk/app/models/cib.rb]: Ignore this

**Comment by tboerger** [comment]
@krig ignore the open violations. Please take a look and merge it when it's fine.

**Comment by krig** [comment]
alright :)

**Pull request merged**